.docker-config/php/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN pecl install mcrypt-1.0.3 && \
 
 RUN curl -sS https://getcomposer.org/installer | php -- \
   --install-dir=/usr/local/bin/ --filename=composer
+RUN composer self-update --1
```

Comment:
At some point we should update this to use the new version, but today doesn't need to be that day.